test/e2e/helpers_test.go

```diff
@@ -89,6 +90,7 @@ func deployKong(ctx context.Context, t *testing.T, env environments.Environment,
 	require.Eventually(t, func() bool {
 		deployment, err = env.Cluster().Client().AppsV1().Deployments(namespace).Get(ctx, "ingress-kong", metav1.GetOptions{})
 		require.NoError(t, err)
+		t.Logf("%d/%d replicas ready", deployment.Status.ReadyReplicas, *deployment.Spec.Replicas)
```

Comment:
It might also be beneficial to log the deployment name since that's not logged anywhere above.